### PR TITLE
Init

### PR DESCRIPTION
**Pull request by melnychukvitaliy**
A lot of things grapped from crispy-cli project toi structure to have mostly the same linting/formatting tools for start

issue is here: #1

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
```
Comment (Bazillius) [inline tsconfig.json]: I guess we should you same tsconfig for all project like https://github.com/WiseEngineering/crispy-cli/blob/master/tsconfig.json and https://github.com/WiseEngineering/crispy-api/blob/master/tsconfig.json

**Comment by Bazillius** [comment]
@melnychukvitaliy It will be a good idea to add Dockerfile for this repo. I think will be better if we will create an issue and assign to @m1g0r

File: tsconfig.json (hunk 1 above)
Comment (melnychukvitaliy) [reply tsconfig.json]: it'll be the different one, we have react here
This one is generated by create-react-app

File: .gitignore [hunk 2]
```diff
@@ -1,61 +1,23 @@
...
-node_modules/
-jspm_packages/
+# production
+/build
 
-# TypeScript v1 declaration files
-typings/
+# misc
+.DS_Store
+.env.local
```
Comment (Bazillius) [inline .gitignore]: Will we use .env files? Looks like we need to review this file. In code above, you add `build` folder to ignore list, did TS build use `build` folder by default? I'm not sure, in `tsconfig.json` you are not set `outDir`. But all this ok, we can review all this later.

Comment (melnychukvitaliy) [reply .gitignore]: we need .env files here for build. 

but we wouldn't use it for production as only static files are required there

About build dir - yes, it's the default path for CRA

**Pull request merged**